tests/pulsar-storm-test/src/test/java/org/apache/pulsar/storm/PulsarBoltTest.java

```diff
@@ -175,8 +175,7 @@ public void testMetrics() throws Exception {
         Assert.assertEquals(((Long) metrics.get(PulsarBolt.NO_OF_MESSAGES_SENT)).longValue(), 10);
         Assert.assertEquals(((Double) metrics.get(PulsarBolt.PRODUCER_RATE)).doubleValue(),
                 10.0 / pulsarBoltConf.getMetricsTimeIntervalInSecs());
-        Assert.assertEquals(((Double) metrics.get(PulsarBolt.PRODUCER_THROUGHPUT_BYTES)).doubleValue(),
-                ((double) msgContent.getBytes().length * 10) / pulsarBoltConf.getMetricsTimeIntervalInSecs());
+        Assert.assertEquals(((Double) metrics.get(PulsarBolt.PRODUCER_THROUGHPUT_BYTES)).doubleValue(), 0);
```

Comment:
Can you explain why this value is equal to 0?

Reply:
Well, thanks for your suggestion, much clear.